### ISSUE TITLE
[text a11y] use a variable Mullish font; add token for 200 weight; darken secondary color

### ISSUE THREAD

**Issue by akdetrick** [narmi/design_system#967]
via @hillarypollak 

### Background
Our secondary text color does not have sufficient contrast below `14px`. We still want to create visual contrast even after darkening the secondary color, so the variable font will allow us to introduce a thin font weight basically for free.

### Mullish change
The best way to introduce new weights is to use the variable version of the google font for Mullish, specifying a weight range rather than explicit weights.

**This will have no impact on download size because Mullish is a variable font**

### Tokens change

Adjust secondary font color to `#6c6c6c`.
Add token for thin font weight (200).
Add helper class for thin font weight (200).
